### PR TITLE
Updated documentation for footer (and main nav)

### PR DESCRIPTION
**Pull request by martinlissmyr**

### CODE REVIEW

**Comment by martinlissmyr** [comment]
ping @lexi @frederfred

**Comment by frederfred** [comment]
My only concern here is that social media links perhaps is something else than generic "External Footer Links". Saving them explicitly as social media links would allow us to render these differently (perhaps as icons) in the future.

👆 also possible to adjust in the future and/or parse links for getting to know the service.

**Comment by martinlissmyr** [comment]
Ok. Good point. I was thinking it was a bit too specific to only allow social media link in proximity to contact. But maybe other links could be something we add later.

File: _data/configuration-navigation.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- field: "Number of visible main navigation items"
```
Comment (lexi) [inline _data/configuration-navigation.yaml:1]: I’m unsure if this is actually needed.

**Comment by martinlissmyr** [comment]
> There is a Facebook ID that is used to connect the website to the Facebook page. In LOM we have this ID as a configuration. One could argue it makes sense to have the actual URL to the page as part of that configuration as well.

Likely. But is that the same thing as saying that the facebook page should be shown in the footer?

File: _data/configuration-navigation.yaml (hunk 1 above)
Comment (martinlissmyr) [reply _data/configuration-navigation.yaml:1]: We could skip it in v1 as least? What do you say @frederfred ? Do we need it for Norr?

Comment (frederfred) [reply _data/configuration-navigation.yaml:1]: I thought we did. I need to see the proposed navigation design until I know for sure.

One way to go forward is to remove it for now and add it when we know for sure that we need it.

Comment (martinlissmyr) [reply _data/configuration-navigation.yaml:1]: Ok, did so now...

**Comment by martinlissmyr** [comment]
> My only concern here is that social media links perhaps is something else than generic "External Footer Links". Saving them explicitly as social media links would allow us to render these differently (perhaps as icons) in the future.

> There is a Facebook ID that is used to connect the website to the Facebook page. In LOM we have this ID as a configuration. One could argue it makes sense to have the actual URL to the page as part of that configuration as well.

These ideas have clear merit but It is VERY opinionated if it is to determine what links should go in the footer after contact. I'm thinking it could also be customer service etc. It also require us to "decide" what social media platforms are "in" etc. If we have a clear use case for knowing, in a semantical sense, which platform is which then I think we should lock it down more... But for now I think it's a bit overengineered. 

So, I'm thinking we'd better go with a more generic "External footer links" for now. If we see a clear case for a more stringent logic around this later on we can add it then.

**Pull request merged**